examples/trusted_information_retrieval/signature.toml

```diff
@@ -0,0 +1,5 @@
+name = "trusted_information_retrieval"
+
+[[signatures.database_proxy]]
```

Comment:
I have added a module name in order to know, which keys correspond to which modules.
Otherwise we would need to check all the modules with all the keys.

I can change module names to module hashes.

Reply:
My question is why is the module name part of the struct? If you just need it as a reminder, it can just be a comment. Also the signature path on line 5 already has the module name in the path itself it seems (though perhaps that will change to a hash too?)

Reply:
It's not just a reminder, it's used by the `oak_runtime` for verifying Wasm modules.
It the `oak_runtime` doesn't have information about which key corresponds to a specific module, then it would need to check all the keys for every single module.

> Also the signature path on line 5 already has the module name ...
Yes, but it's only happened, because we have chosen the examples to be in this directory. Not all applications will contain such an information.

Reply:
Another problem with having the name as map key here is that, IIUC, you can still only have at most one signature per module, which we discussed is too limiting.

If we had the hash here, we could have a comment that says "database_proxy" in the TOML file, and I think that will serve the purpose. The runtime will not use the name for anything, and signing modules does not involve knowing their name, or their name being consistent.

Reply:
I've already updated this, it accepts multiple signatures per module.
In this file double square brackets `[[` mean that this is an array.
So it may looks like this:
```toml
[[signatures.database_proxy]]
...

[[signatures.database_proxy]]
...
```

Reply:
I'll change it into a hash, and use comments for module names.

Reply:
ah got it, thanks, I wasn't familiar with the syntax! still the other points stand :) maybe we can use the hash as part of the array, but have a comment with the name. basically I don't think the runtime nor the signing process should even know about the name. WDYT?